core/chains/evm/headtracker/head_tracker_test.go

```diff
@@ -320,7 +320,8 @@ func TestHeadTracker_ResubscribeOnSubscriptionError(t *testing.T) {
 			},
 			func(ctx context.Context, ch chan<- *evmtypes.Head) error { return nil },
 		)
-	ethClient.On("HeadByNumber", mock.Anything, mock.Anything).Return(cltest.Head(0), nil)
+	ethClient.On("HeadByNumber", mock.Anything, mock.Anything).Return(cltest.Head(0), nil).Once()
```

Comment:
would this fail as it is not being called?

Reply:
`HeadByNumber` is still being called during HeadTracker startup to fetch the initial head